src/componentes/NombrePersona.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import useNombrePersona from '../hooks/useNombrePersona.js';
+
+const NombrePersona = (props)=>{
+    const {nombre,caracteres,mayusNombre} = useNombrePersona(props.name);
```

Comment:
hello @mrHann69 using the props **name** in this way is going to work perfectly BUT I would prefer to destructure it